### PR TITLE
Add doublemetaphone recipe

### PR DESCRIPTION
**Pull request by proinsias**
- [ ] Include license file from DoubleMetaphone once it has been provided – see my [issue](https://github.com/datamade/doublemetaphone/issues/1).

### CODE REVIEW

File: recipes/doublemetaphone/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (jakirkham) [inline recipes/doublemetaphone/build.sh]: Please drop this file. No need as `script` does this for us.

File: recipes/doublemetaphone/bld.bat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
+if errorlevel 1 exit 1
+
+:: Add more build steps here, if they are necessary.
```
Comment (jakirkham) [inline recipes/doublemetaphone/bld.bat]: Please drop this file. No need as `script` does this for us.

File: recipes/doublemetaphone/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win and linux]
```
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Is this just `osx`? If so, this should be `# [win or linux]` or better yet `# [not osx]`. Currently this `skip` is getting dropped.

Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Also why is this OS X only just out of curiosity?

File: recipes/doublemetaphone/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
```
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Please drop these `conda skeleton` comments.

File: recipes/doublemetaphone/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+about:
+  license: MIT License
```
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Please shorten to `MIT`.

Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Please open an issue upstream asking for a license file and its inclusion in `MANIFEST.in`. Also please add a comment here to include it.

File: recipes/doublemetaphone/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
```
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Please drop this comment.

File: recipes/doublemetaphone/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win and linux]
+
+requirements:
+  build:
```
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Please add `toolchain` as this is compiled code.

File: recipes/doublemetaphone/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Looks like it has Cython. So could we please add `cython` here?

File: recipes/doublemetaphone/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: Hmm...says [POSIX]( https://github.com/datamade/doublemetaphone/blob/7688cb90b5fd1427f788cc0672183ccea94fc45d/setup.py#L19 ) upstream. Why would it not work on Linux then?

**Comment by jakirkham** [comment]
Thanks @proinsias. A couple of things to fix up. None of them should be too bad. Some of them should help with the CI failures.

Also please merge with `master` here and delete any recipes that you are not adding in this PR (except the `example`).

**Comment by proinsias** [comment]
I've pushed an update with most of the changes from @jakirkham – thanks for the help!

Still to do:
- [ ] Include license file from DoubleMetaphone once it has been provided

File: recipes/doublemetaphone/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - doublemetaphone
+
+about:
+  license: MIT
+  # license_file: LICENSE # When added to MANIFEST.in – https://github.com/datamade/doublemetaphone/issues/1
```
Comment (jakirkham) [inline recipes/doublemetaphone/meta.yaml]: xref: https://github.com/datamade/doublemetaphone/issues/1
xref: https://github.com/datamade/doublemetaphone/pull/2

**Comment by jakirkham** [comment]
Thanks @proinsias.

**Pull request merged**